### DISCUSSION TITLE
App signed, sealed and delivered

### DISCUSSION THREAD

**Discussion by mislavmatijevic** [foivz/pop-app#36] [Announcements]
So, right now the app is done. Everything works and the clone is pretty clean. Thus, project #106 is closed.

Next big step (for the next year) is recreating the current God awful pure-PHP """RESTful""" backend abomination our predecessors put together and left us with into something acceptable.
Before we start working on that, the only thing left to decide is whether to use PHP or not and whether to create a new repo for that or not.

I suggest going with PHP for the sake of simplicity of the refactoring process and creating a new repository for splitting the two parts of the project apart.

**Comment by jfletcher20** [comment]
Agreed, one big flaw with the latest issues #38 and #37 is we're go to implement them as-is, but I think it would be better maybe if we do the backend changes first, then continue with the app, just so we don't end up having to backtrack and retrace our steps when we redo the backend. Whenever you're ready I can check in at the office to talk about it.

I think we should use PHP (Laravel). Among other things, it'd be nice to get better at it.
